cfgov/templates/wagtailadmin/css/bureau-structure.css

```diff
@@ -0,0 +1,21 @@
+
+.blockname-bureau_structure li {
+    padding-bottom: 10px;
+}
+
+@media only screen and (min-width: 800px) {
+  .blockname-bureau_structure
+  .sequence-member
+  .sequence-member-inner {
+    min-width: 500px;
+  }
+
+  .blockname-bureau_structure li[id*="office_of_the_director"],
+  .blockname-bureau_structure li[id*="divisions"] {
+    border: 1px solid #e6e6e6 !important;
+    margin: 5px 5px 5px 10px;
+    padding: 10px;
+    border-radius: 6px;
+    min-width: 440px;
+  }
+}
```

Comment:
In general, I think these customizations are fine for now. We should make a more concerted effort to improve the styling of the Wagtail editor at some point. Lots of questionable decisions in there (especially the decreasing widths as you go deeper nested fields).